analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/SymbolReferenceAnalyzerTest.cs

```diff
@@ -255,6 +258,16 @@ private VerifierBuilder CreateBuilder(ProjectType projectType, params string[] f
                 .WithProtobufPath(@$"{testRoot}\symrefs.pb");
         }
 
+        private static void VerifyReferences(IReadOnlyList<SymbolReferenceInfo.Types.SymbolReference> references,
+                                             int expectedDeclarationCount,
+                                             int assertedDeclarationLine,
+                                             params int[] assertedDeclarationLineReferences)
+        {
+            references.Where(x => x.Declaration != null).Should().HaveCount(expectedDeclarationCount);
+            var declarationReferences = references.Single(x => x.Declaration.StartLine == assertedDeclarationLine).Reference;
+            declarationReferences.Select(x => x.StartLine).Should().BeEquivalentTo(assertedDeclarationLineReferences);
```

Comment:
`declarationReferences` is only used ones, so a I think we can inline the variable and use fluent notation.